config/routes.rb

```diff
@@ -5,7 +5,8 @@
 
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
```

Comment:
I like this versioning strategy. I've used it in the past and it definitely gets the job done. :+1: Other people may have differing opinions on this style, though. Just something to keep in mind.